hspy_bundle/NSIS_installer_script.nsi

```diff
@@ -307,17 +307,18 @@ SectionIn RO
 	File /r "${WINPYTHON_PATH}\*"
 	${If} $InstMode = 2
 	; Create right-click context menu entries for Hyperspy Here
-		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython Command Prompt.exe" install_hyperspy_here & exit"'
+		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython Command Prompt.exe" jupyter_context-menu_add & exit"'
 	Sleep 3000
 	${EndIf}
 
 	${If} $InstMode <> 1
 	; Create StartMenu shortcuts
 		CreateDirectory "$SMPROGRAMS\${APPNAME}"
-		CreateShortCut "$SMPROGRAMS\${APPNAME}\${APPNAME}.lnk" "${APP_INSTDIR}\${PYTHON_FOLDER}\Scripts\hyperspy.bat"
+		CreateShortCut "$SMPROGRAMS\${APPNAME}\WinPython prompt.lnk" "${APP_INSTDIR}\WinPython Command Prompt.exe"
+		CreateShortCut "$SMPROGRAMS\${APPNAME}\IPython.lnk" "${APP_INSTDIR}\${PYTHON_FOLDER}\Scripts\ipython.exe"

```

Comment:
Either way you can get the output from the "artifacts" tab: https://ci.appveyor.com/project/to266/hyperspy/build/job/gd1btjhmrnw6wh05/artifacts